src/adapter/src/catalog/open.rs

```diff
@@ -478,6 +478,23 @@ impl Catalog {
                 match builtin {
                     Builtin::Log(log) => {
                         let oid = catalog.allocate_oid()?;
+                        let mut acl_items = vec![rbac::owner_privilege(
+                            mz_sql::catalog::ObjectType::Source,
+                            MZ_SYSTEM_ROLE_ID,
+                        )];
+                        match log.sensitivity {
+                            Sensitivity::Public => {
+                                acl_items.push(rbac::default_builtin_object_privilege(
+                                    mz_sql::catalog::ObjectType::Source,
+                                ));
+                            }
+                            Sensitivity::SuperuserAndSupport => {
+                                acl_items.push(rbac::support_builtin_object_privilege(
+                                    mz_sql::catalog::ObjectType::Source,
+                                ));
+                            }
+                            Sensitivity::Superuser => {}
+                        }
```

Comment:
Not really that important, but there's probably a way to condense these all by adding something like the following before the match statement:

```Rust
                        let object_type = ObjectType::From(&builtin);
                        let mut acl_items = vec![rbac::owner_privilege(
                            object_type,
                            MZ_SYSTEM_ROLE_ID,
                        )];
                        match builtin.sensitivity() {
                            Sensitivity::Public => {
                                acl_items.push(rbac::default_builtin_object_privilege(
                                    object_type,
                                ));
                            }
                            Sensitivity::SuperuserAndSupport => {
                                acl_items.push(rbac::support_builtin_object_privilege(
                                    object_type,
                                ));
                            }
                            Sensitivity::Superuser => {}
                        }
```

Reply:
yeah, the fact that this doesn't apply to functions/indexes/types is exactly why I decided not to do it like that in the first place.